### ISSUE TITLE
Streaming Issues

### ISSUE THREAD

**Issue by chrootnewb** [dnschneid/crouton#3498]
```
Please paste the output of the following command here: sudo edit-chroot -all
```

**Comment by chrootnewb** [comment]
my device is the acer r11

**Comment by DennisLfromGA** [comment]
@chrootnewb,

> Did a powerwash and everything is working fine again. how can i solve this ?

Solve what? 
It sounds like it's working...

If you're still having problems give us some more details -
1. Show us the output of: `sudo edit-chroot -all`
2. What videos don't work in CrSO & your chroot - links please?

-**DennisL**

**Comment by chrootnewb** [comment]
Hi Dennis sorry about not providing enough details. What happened was after
I installed crouton, I tried watching Netflix, then youtube, then I tried
streaming a video from another site. I was getting the same issue being I
would press play and the video would seem to start buffering, but wouldn't
proceed past the first frame. On both croot and chrome, so I went back to
chrome did a powerwash, re-enabled os verification, then proceeded with the
crouton installation process. After I was finished the second time the
streaming issues stopped. I'm new to this and I'm sorry if I was supposed
to close the thread on GitHub or something. But no longer having problems.
Thanks for the help though!
